my-app/src/todoApp.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import Add from './components/add.js'
+import Search from './components/search.js'
+
+class TodoApp extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      items: [],
+      searchitems : []
+    };
+  }
+
+  deleteElement(itemIndex) {
+    this.state.items.splice(itemIndex, 1);
```

Comment:
Never mutate the state variable, Assign it to new copy and do manipulation on new copy